### DISCUSSION TITLE
Questions about average response time

### DISCUSSION THREAD

**Discussion by zhougy1989** [naver/ngrinder#876] [Q&A]
I found a defect. When two interface functions are put into a script, the response time in the report is the average value of the two interfaces. Shouldn't this be the two functions you want to add? I want to ask where the code for calculating the average response time is, and I'll adjust it myself

![image](https://user-images.githubusercontent.com/50607317/167748680-885f9f02-db55-4b82-abc9-ac2558d9c66d.png)

I don't think this picture is reasonable
can you check it?
@junoyoon

**Comment by imbyungjun** [comment]
@zhougy1989 
You mean the `Mean Test Time` graph on detail report right?

You should change below codes. You should draw the chart `mean-time-chart` as you want rather than invoking `drawChart()` method
https://github.com/naver/ngrinder/blob/58b6033429fe05b40ac4cfedee8747a97da6c97f/ngrinder-frontend/src/js/components/perftest/report/menu/PerfTest.vue#L113-L122

**Reply by zhougy1989** [reply]
thks,and Where is the time calculation method:Mean_Test_Time_(ms)
![image](https://user-images.githubusercontent.com/50607317/167971507-e9304cfa-a0cd-41ee-b331-f0a074a000a3.png)
Where is the calculation method of the red marked parameters in the above figure

@junoyoon ，
Sorry to bother you
